### PR TITLE
Enable Google analytics

### PR DESCRIPTION
**Pull request by nickmwangemi**

### CODE REVIEW

**Review by longhotsummer — COMMENTED** [review]
Is the necessary javascript being injected into the template already?

**Comment by nickmwangemi** [comment]
> Is the necessary javascript being injected into the template already?

In the `base.html` template?

File: peachjam/templates/peachjam/layouts/main.html [hunk 1]
```diff
@@ -17,6 +17,15 @@
...
     });
     </script>
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id={{ GOOGLE_ANALYTICS_ID }}"></script>
+    <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag(){dataLayer.push(arguments);}
+      gtag('js', new Date());
+
+      gtag('config', 'UA-34245896-1');
```
Comment (longhotsummer) [inline peachjam/templates/peachjam/layouts/main.html:27]: This must be `{{ GOOGLE_ANALYTICS_ID }}`

File: peachjam/templates/peachjam/layouts/main.html [hunk 2]
```diff
@@ -17,6 +17,15 @@
         });
     });
     </script>
+    <!-- Global site tag (gtag.js) - Google Analytics -->
```
Comment (longhotsummer) [inline peachjam/templates/peachjam/layouts/main.html:20]: Only include this snippet if `GOOGLE_ANALYTICS_ID` is not None

File: peachjam/templates/peachjam/layouts/main.html [hunk 3]
```diff
@@ -17,6 +17,17 @@
...
     </script>
+    {% if GOOGLE_ANALYTICS_ID %}
+      <!-- Global site tag (gtag.js) - Google Analytics -->
+      <script async src="https://www.googletagmanager.com/gtag/js?id={{ GOOGLE_ANALYTICS_ID }}"></script>
+      <script>
+        window.dataLayer = window.dataLayer || [];
+        function gtag(){dataLayer.push(arguments);}
+        gtag('js', new Date());
+
+        gtag('config', {{ GOOGLE_ANALYTICS_ID}});
```
Comment (longhotsummer) [inline peachjam/templates/peachjam/layouts/main.html:28]:
```suggestion
        gtag('config', '{{ GOOGLE_ANALYTICS_ID}}');
```

**Review by longhotsummer — APPROVED** [review]

**Pull request merged**